### DISCUSSION TITLE
Using openWakeWord in Openpilot

### DISCUSSION THREAD

**Discussion by jakethesnake420** [dscripka/openWakeWord#104] [Show and tell]
I have spent the last few days adding the openWakeWord project to OpenPilot! What a perfect match. See my PR here https://github.com/commaai/openpilot/pull/31010

**Comment by dscripka** [comment]
This is very cool! Thank you for working to add openWakeWord into OpenPilot! If the Comma 3 has enough power to run the OpenPilot models as well as openWakeWord, I can definitely see the potential for some interesting applications.
